README.md

```diff
@@ -221,3 +221,12 @@ let needToResetLogin = token.isExpired;
 // The original JWT string
 let jwt = token.token;
 ```
+
+## Build
+
+### Releases
+
+To cut a release:
+
+1. Bump the version in `package.json`, and merge that to `main`
+2. Create a GitHub release. The presence of the tag will trigger a CircleCI run that deploys to npm.
```

Comment:
Actually, might be a good idea to put this in a separate markdown doc in the repo (e.g. RELEASES.md) -- npm displays the README on the package homepage, and this isn't really relevant for users

Reply:
(not super important either way)